apps/pair_align/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ message (STATUS "Configuring apps/pair_align")
 
 set (SEQAN_APP_VERSION "1.3.7")
 
+if (SEQAN_TRAVIS_BUILD)
```

Comment:
Is it wise to always disable it? 
Maybe if we can detect the compiler version used, we can disable it based on the used `CXX` environment variable.
All other builds usually run through.

Reply:
But bringing down the build times in general is a good idea. Also we might add new tests that then again require to deactivate something else..